### ISSUE TITLE
Practical 6 Aim:- Perform the logistic regression on the given data warehouse data using Python

### ISSUE THREAD

**Issue by RenuSam6964** [RenuSam6964/TY#1]
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import accuracy_score,classification_report,confusion_matrix
import matplotlib.pyplot as plt

data={'Feature1':[1.2,4.5,9.6,8.3,5.0,4.5],
      'Feature2':[7.1,4.5,8.6,9.5,3.8,7.5],
      'Target':[0,0,1,1,1,0]}
Df=pd.DataFrame(data)

x=Df[['Feature1','Feature2']]
y=Df['Target']

x_train,x_test,y_train,y_test=train_test_split(x,y,test_size=0.2,random_state=42)

model =LogisticRegression(max_iter=1000)
model.fit(x_train,y_train)

Y_pred=model.predict(x_test)

accuracy=accuracy_score(y_test,Y_pred)
print(f'Accuracy:{accuracy:.3f}')
print('classifiaction Report:')
print(classification_report(y_test,Y_pred))
print('Confusion Matrix')
print(confusion_matrix(y_test,Y_pred))

plt.title('Logistic Regression')
plt.scatter(x_test['Feature1'],x_test['Feature2'],c=Y_pred)
plt.xlabel('Feature 1')
plt.ylabel('Feature 2')
plt.show()
